src/pudl/metadata/resources/ferc1.py

```diff
@@ -387,6 +387,30 @@
         "etl_group": "glue",
         "field_namespace": "ferc1",
     },
+    "utility_plant_summary_ferc1": {
+        "description": "Summary of utility plant and accumulated provisions for depreviation, amortization and depletion of utilty plant assets reported annually at the end of the report year.",
+        "schema": {
+            "fields": [
+                "utility_id_ferc1",
+                "report_year",
+                "utility_type",
+                "utility_type_other",
```

Comment:
What's going on with the `utility_type_other` column in here? Is it like, a description of the "other" type that's been reported in `utility_type`?

Reply:
yes exactly. This column only shows up in the DBF data, but it is meant to specific wtf the "other" utility type actually is. I considered applying `categorize_strings` to it but there is so much information in here that could not be categorized easily.

```
water                     1266
steam                      651
common                     189
bhsc                       184
steam heat                 163
bhuhc                       88
steam heating               85
capital lease               64
steam distribution          60
propane                     55
gas                         53
thermal                     50
propane gas                 50
microwave                   48
elm road                    48
bhuh                        48
bh service co               48
bh utility holding          48
energy delivery             45
wastewater                  38
transit                     36
peaking plant               32
general plant-steam         30
bhsc allocation             29
coal mine/reserves          28
steam services              27
oth.ut.plt.acct.118         24
ind. steam                  24
wccf                        24
other utility plant         24
non-utility property        22
industrial steam            21
telephone                   20
liquid fuels                18
nonutility                  17
oth.ut.plt.act.118          16
utility holding co          16
non-utility                 12
other utility               12
newy                        10
gen. plant steam            10
non utility                 10
gen. plnt - steam           10
themal                      10
general plant-stm           10
energy services             10
______water_____             9
coal mine                    9
fresh start acct adj         9
coach                        9
capital leases               8
(bhsc)                       8
(bhuhc)                      8
rou operating lease          8
service comp alloc           8
rou, operating lease         8
serv comp alloc              8
bh service co.               8
west campus                  8
Name: utility_type_other, dtype: int64
```

Reply:
**bhuh** indeed.

Might this data be more appropriately treated as an unstructured (sub-)table, since it will be meaningless to compare an `Other1` record from one filing with another, while presumably the electric, gas, common and total values would be more comparable?

Reply:
despite the 3 "other" utility types being less informative about what type of utility they are, this data still has all of the other categories and structure that this table includes. i would no removing it.